configs/karaf/fcrepo_camel_toolbox.script

```diff
@@ -1,3 +1,5 @@
-feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.6.2/xml/features
+feature:repo-add camel 2.18.1
+feature:repo-add activemq 5.14.0
+feature:repo-add mvn:org.fcrepo.camel/toolbox-features/4.7.0/xml/features
```

Comment:
If we install Alpaca first, we don't need these feature declarations.  Alpaca's features.xml will take care of it for us and you can delete these three lines.